emails/mailer.js

```diff
@@ -12,11 +12,20 @@ module.exports = function init (db) {
     if (key.indexOf('msg!') !== 0) return
     if (!value.route) return
     if (!value.route.email) return
+    if (!value.headers['X-Spam-Flag']) return
```

Comment:
Check for property existance
```js
value.headers.hasOwnProperty('X-Spam-Flag')
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty